src/content/self-hosted/install/releases.mdx

```diff
@@ -5,6 +5,33 @@ sidebar_label: Release Notes
 id: releases
 ---
 
+## 1.14.0
+
+[DATE]
+
+This version is compatible with Kubernetes versions 1.24 to 1.27.
+
+### Deprecatation Notice
+
+- As of this versions deploying helm charts through the Okteto UI has been deprecated.
```

Comment:
We are nor deprecating it, we are removing it (we no longer support the `application` key). We should probably document it as Breaking change as any potential instance using the key will no longer be able to deploy charts from the helm catalog.

@rlamana @viddo What do you think is the best way of communicating this?

Reply:
@bigash42 FYI in case we need to prepare any additional comms

Reply:
+1, I'd suggest:
```suggestion
- Helm charts catalog is no longer available in the Okteto UI. For alternative approach to deploy Helm charts see https://www.okteto.com/blog/making-your-helm-packaged-applications-ready-for-cloud-native-development-with-okteto/
```

Reply:
Just a note. This wasn't available by default in SH since some time ago. What we completely removed is the possibility of enabling it (https://www.okteto.com/docs/self-hosted/administration/configuration/#applications).

Mentioning it because referring only to Okteto UI might be misleading. Also, we have to be very clear this is nothing to do with the Catalog feature.